Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -891,7 +891,8 @@ def SwapKeepAndDrop(l):
         'keep Phase2TrackerDigiedmDetSetVector_mix_*_*',
         'keep *_TTClustersFromPhase2TrackerDigis_*_*',
         'keep *_TTStubsFromPhase2TrackerDigis_*_*',
-        'keep *_TrackerDTC_*_*'
+        'keep *_TrackerDTC_*_*',
+        'keep *_*_Level1TTTracks_*'
```

Comment:
Are you sure you need to add *_*_Level1TTTracks_*? I think the L1 tracks are already included in the FEVTEventContent (they are used for the upcoming re-reco of the HLT TDR samples).

Reply:
I am not completely sure, no. At the time of the original PR, it was needed, or else no tracks were saved. From digging through the EventContent, it appears the tracks are saved in phase2_muon, but not in phase2_tracker, but both are run in the Phase2 chain. I can definitely remove this and see if things still run well.